server/app/api.py

```diff
@@ -1683,6 +1684,42 @@ def score(self, obj, user, data):
 
         return score
 
+
+class GradeAPI(APIResource):
```

Comment:
I don't think `GradeAPI` needs to exist, because we have `SubmissionAPI.score` already.